code/modules/bitrunning/event.dm

```diff
@@ -7,16 +7,12 @@
 	category = EVENT_CATEGORY_INVASION
 	description = "Causes a short term antagonist to spawn in the virtual domain."
 	dynamic_should_hijack = FALSE
-	max_occurrences = 5
 	min_players = 1
+	max_occurrences = 0
 	typepath = /datum/round_event/ghost_role/bitrunning_glitch
-	weight = 10
-	/// List of active servers to choose from
-	var/list/obj/machinery/quantum_server/active_servers = list()
-	/// List of possible antags to spawn
-	var/static/list/possible_antags = list(
-		ROLE_CYBER_POLICE,
-	)
+	weight = 100
+	/// List of servers on the station
+	var/list/active_servers = list()
```

Comment:
is this list managed?

Reply:
Wdym managed in this context?

Reply:
is it a list of objects that we are holding references to

Reply:
it's cut as soon as the event fires. i can make them weakrefs too